onnxruntime/core/providers/cpu/tensor/slice.h

```diff
@@ -11,27 +11,18 @@ class SliceBase {
  protected:
   SliceBase(const OpKernelInfo& info) {
     has_axes_ = info.GetAttrs("axes", axes_).IsOK();
-
-    ORT_ENFORCE(info.GetAttrs("starts", starts_).IsOK(), "Invalid 'starts' attribute value");
-    ORT_ENFORCE(info.GetAttrs("ends", ends_).IsOK(), "Invalid 'ends' attribute value");
-
-    if (has_axes_) {
-      if (axes_.size() > starts_.size())
-        ORT_THROW("'axes' has more entries than the 'starts' attribute holds");
-      if (axes_.size() > ends_.size())
-        ORT_THROW("'axes' has more entries than the 'ends' attribute holds");
-    }
+    info.GetAttrs("starts", starts_);
```

Comment:
They are inputs, not attributes.

Reply:
Thanks, but it's been corrected already - are you looking at removed line?